tsp-output/@azure-tools/typespec-csharp/src/Generated/OpenAIClient.cs

```diff
@@ -13,6 +13,9 @@ namespace OpenAI
     /// <summary> The OpenAI service client. </summary>
     public partial class OpenAIClient
     {
+        protected ApiKeyCredential KeyCredential => _keyCredential;
```

Comment:
One problem with this is that if the client doesn't use api key auth, what does this do?